### ISSUE TITLE
Could not reproduce pretrained model

### ISSUE THREAD

**Issue by jpsml** [AaltoVision/deep-speed-constrained-ins#1]
I have followed the instructions for downloading and preparing training data and tried to train a model by changing `load_model` to `False` and both `save_model` and `train_model` to `True` in DCI-training-0.0.2.py. I also needed to change two lines because I am using pytorch 1.0.0.

I have changed this line:
`acc_loss +=np.sum(loss.data[0])`

to this:
`acc_loss +=np.sum(loss.item())`

And I have changed this line:
`val_loss +=np.sum(loss.data[0])`

to this:
`val_loss +=np.sum(loss.item())`

However, I am getting the following results:

![figure_1](https://user-images.githubusercontent.com/13179144/50194030-71b49980-037c-11e9-98ae-a9e1b8a8b293.png)
![figure_2](https://user-images.githubusercontent.com/13179144/50194031-71b49980-037c-11e9-9de6-71f5f09d1dff.png)
![figure_3](https://user-images.githubusercontent.com/13179144/50194032-71b49980-037c-11e9-9940-ee9216fa99b0.png)
![figure_4](https://user-images.githubusercontent.com/13179144/50194033-724d3000-037c-11e9-92f6-e6bef3bcfac8.png)
![figure_6](https://user-images.githubusercontent.com/13179144/50194034-724d3000-037c-11e9-95e8-0368fd58d266.png)
![figure_7](https://user-images.githubusercontent.com/13179144/50194035-724d3000-037c-11e9-8291-b993a88390cf.png)
![figure_8](https://user-images.githubusercontent.com/13179144/50194036-72e5c680-037c-11e9-8866-747fbee0857f.png)

The final values for loss and validation loss were 2.2964887819458952 and 2.3019133200951383, respectively. Is this expected? Should I change any other parameter (like number of epochs) in order to reproduce the pretrained model?

I am using Ubuntu 18.04.1 LTS, Python 2.7.15rc1 and pandas 0.23.4.

Best regards.
